Packs/CommonScripts/Scripts/ExtractAttackPattern/ExtractAttackPattern_test.py

```diff
@@ -0,0 +1,24 @@
+from CommonServerPython import *
+
+
+ATTACK_PATTERN_OBJ = [{"name": "Two-Factor Authentication Interception"}]
+
+
+def test_fetch_indicators(mocker):
```

Comment:
please add  a tests for the cases: 
1. integration is not configured
2. the ids sent to the integration new not found.
3. only one id was passed the the script

Reply:
done